src/main/java/com/celements/javascript/JsFileEntry.java

```diff
@@ -67,6 +67,11 @@ public boolean isValid() {
     return !jsFileUrl.isEmpty();
   }
 
+  public boolean isModule() {
+    String fileName = getFilepath().split("[?$]")[0];

```

Comment:
since `getFilePath` misleadingly doesn't return a path (as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986#section-3.3)) but may also contain a query parts, it's better to use an existing parser instead of custom parsing: 
```suggestion
    String fileName = UriBuilder.fromUri(getFilepath()).build().getPath();
```
This also works for empty input strings.